README.md

```diff
@@ -1,12 +1,54 @@
+[![CharmHub Badge](https://charmhub.io/httprequest-lego-provider/badge.svg)](https://charmhub.io/httprequest-lego-provider)
+[![Publish to edge](https://github.com/canonical/httprequest-lego-provider/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/httprequest-lego-provider/actions/workflows/publish_charm.yaml)
+[![Promote charm](https://github.com/canonical/httprequest-lego-provider/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/httprequest-lego-provider/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
+
 # HTTPRequest Lego Provider
 
-A Django application implementing [HTTPRequest Lego Provider](https://go-acme.github.io/lego/dns/httpreq/).
+A Juju charm deploying and managing [HTTP request Lego provider as defined by ACME](https://go-acme.github.io/lego/dns/httpreq/)
+to manage DNS records. It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
+offerings.
 
-## Project and community
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more. For Charmed HTTP request Lego provider, this includes:
+  - Scaling
+  - Integration with PostgreSQL
+
+For information about how to deploy, integrate, and manage this charm, see the Official [HTTP request Lego provider Documentation](https://charmhub.io/httprequest-lego-provider/docs).
+
+
+## Get started
+
+You can follow the tutorial [here](https://charmhub.io/httprequest-lego-provider/docs/tutorial-deploy-the-deploy-the-httprequest-lego-provider-charm-for-the-first-time).
+
+### Basic operations
 
-The HTTPRequest Lego Provider is a member of the Ubuntu family. It's an open source
-project that warmly welcomes community projects, contributions, suggestions,
-fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+The following actions are available for this charm:
+  - allow-domains: Grant user access to domains.
+  - create-superuser: Create a new Django superuser account.
+  - create-user: Create a user for the services that will be requesting the domains. If it exists, the password will be updated.
+  - list-domains: List the domains an user has access to.
+  - revoke-domains: Revoke user access to domains.
+  - rotate-secret-key: Rotate the secret key. Users will be forced to log in again. This might be useful if a security breach occurs.
```

Comment:
I don't think there's much point in just repeating the action descriptions (you could just refer to the Charmhub actions page instead). I think this section is more about giving juju cli examples for basic day "n" operations. So maybe pick one or two of the most important ones and give command line examples. @erinecon wdyt?

Reply:
While I don't have a strong opinion about listing all the actions in the README, I think that mentioning the most important actions is a smart approach for maintainability purposes -- then you won't need to update the README every time there's a new action added to the charm. 

Similarly with configurations, if there are any required configurations needed for the charm to work, then it's worth listing in this section as well. 

If you trim down this section, then please include a link to the actions tab in the Charmhub documentation.